### DISCUSSION TITLE
Fully disable italics

### DISCUSSION THREAD

**Discussion by satoqz** [kovidgoyal/kitty#6515] [Q&A] [answered]
I'd like to configure my font such that no italics are ever displayed, i.e. `italic_font` is displayed as the normal font and `bold_italic_font` is equal to `bold_font`.

I've tried a few things to no avail, the italics won't go away:

```kitty
font_family      IBM Plex Mono
font_bold        IBM Plex Mono Bold
font_italic      IBM Plex Mono Regular
font_bold_italic IBM Plex Mono Bold
```

Any suggestions?

**Comment by chdiza** [comment, marked as answer]
I want the same thing except for "bold".

Last time I checked, if the font family has a bold variant installed, kitty will use it even if in your kitty.conf you declare `font_bold  [the-non-bold-variant-of-the-font]`.  It's annoying but the workaround is to install font that doesn't even have bold or italic variants in it.  E.g., I have installed only non-bold and non-italic variants of Input Mono and Consolas.

I suspect that what I found to be the case for bold is also behind your trouble with italics.

(I would desperately love to banish italics *and* bold in my kitty.conf.)
